tests/integration_tests/functional/test_feat_parity.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Tests for the verifying features exposed by CPUID and MSRs by various CPU templates."""
+
+import pytest
+
+from conftest import _test_images_s3_bucket
+from framework.artifacts import ArtifactCollection, ArtifactSet
+from framework.matrix import TestMatrix, TestContext
+from framework.builder import MicrovmBuilder
+import framework.utils_cpuid as cpuid_utils
+import host_tools.network as net_tools
+
+
+# All existing CPU templates available on Intel
+INTEL_TEMPLATES = ["C3", "T2", "T2S"]
+# All existing CPU templates available on AMD
+AMD_TEMPLATES = []
+# All existing CPU templates
+ALL_TEMPLATES = INTEL_TEMPLATES + AMD_TEMPLATES
+# CPU templates designed to provide instruction set feature parity
+INST_SET_TEMPLATES = []
+
+
+def get_supported_templates():
+    """
+    Return the list of CPU templates supported by the platform.
+    """
+    vendor = cpuid_utils.get_cpu_vendor()
+    if vendor == cpuid_utils.CpuVendor.INTEL:
+        return INTEL_TEMPLATES
+    if vendor == cpuid_utils.CpuVendor.AMD:
+        return AMD_TEMPLATES
+    return []
+
+
+SUPPORTED_TEMPLATES = get_supported_templates()
+
+
+def intersection(lst1, lst2):
+    """
+    Return the list that is the intersection of two lists.
+    """
+    lst3 = [value for value in lst1 if value in lst2]
+    return lst3
```

Comment:
You could use `set()s` instead, which have an intersection method.

Reply:
done